src/eredis_client.erl

```diff
@@ -368,7 +370,8 @@ safe_send(Pid, Value) ->
     try erlang:send(Pid, Value)
     catch
         Err:Reason ->
-            error_logger:info_msg("eredis: Failed to send message to ~p with reason ~p~n", [Pid, {Err, Reason}])
+            ?LOG_NOTICE("eredis: Failed to send message to ~p with reason ~p~n",
```

Comment:
```suggestion
            ?LOG_NOTICE("Failed to send message to ~p with reason ~p",
```
Is there a reason to keep the newline in each log? Seems to be removed when using a formatter template anyway.